### PR TITLE
Feature/firewx fixes

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

Removing hardcoded paths to perry.shafran and replacing them with pulls from the parallel.  

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Run the stats code to ensure that the removal of the extra parm file doesn't affect the code.
Run the plots code to see that the correct hrrr/namnest files are being copied in.  

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

ASAP

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
@PerryShafran-NOAA , I have cloned your fork and checked out branch `feature/firewx_fixes`.

**Comment by ShelleyMelchior-NOAA** [comment]
I have generated stats for cyc=04. Stats all compare exactly w/ emc.vpppg stats. Please review.
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_nam_firewxnest_grid2obs_stats_00.o96155464`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam/atmos.20230906/nam_firewxnest/grid2obs`

I also ran plots. The tar files compares well w/ emc.vpppg. Please reivew
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_nam_firewxnest_grid2obs_plots.o96155523`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/plots/cam/atmos.20230906`

**Comment by ShelleyMelchior-NOAA** [comment]
@PerryShafran-NOAA  -- I am not remembering the status of this PR. I didn't see any formal feedback, but I do see some code updates, so I took the liberty to re-run stats and plots. Please review.

STATS
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_nam_firewxnest_grid2obs_stats_00.o96430*`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam/atmos.20230909/nam_firewxnest/grid2obs/`

PLOTS
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_nam_firewxnest_grid2obs_plots.o96431975`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/cam/atmos.20230908`

**Comment by PerryShafran-NOAA** [comment]
@ShelleyMelchior-NOAA OK, for stats, I see the SENDCOM stuff in here, DATA is set correctly, stat files are properly being written from stmp to the com directory.  

I've been told that I need to remove FIXevs from the driver script, so I'll need to do that.  I still see some verbosity statements that shouldn't be in there.  

Other than that, the stats run looks good.

**Comment by ShelleyMelchior-NOAA** [comment]
Awesome! I will look for those fixes and then do the formal code review.

**Comment by PerryShafran-NOAA** [comment]
Think that some of these fixes might be worth another test.  I'd feel better if they were.

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA It looks like your symbolic linked /fix directory may have made it into this PR? I'm not 100% sure. Can we make sure that gets removed? I'll let you work with Shelley on the rest. Thanks! 

https://github.com/NOAA-EMC/EVS/pull/229/commits/799a7167af1232525e45781b92d2a9ec86026d99#diff-1c6e6c4c02e55178e85890fc9bbed4ce046415ec8122bf38f711b779184ae2a0R1

File: fix
Comment (ShelleyMelchior-NOAA) [inline fix:1]: Let's remove this link from your feature branch. I will handle the symbolic link set up in my local checkout.

File: dev/drivers/scripts/cam/plots/jevs_cam_nam_firewxnest_grid2obs_plots.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 export model=evs
 
-export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS_firewx_fixes/EVS
```
Comment (ShelleyMelchior-NOAA) [inline dev/drivers/scripts/cam/plots/jevs_cam_nam_firewxnest_grid2obs_plots.sh]: And you konw the drill here ... revert to ${USER} for HOMEevs.

**Comment by ShelleyMelchior-NOAA** [comment]
I re-ran stats and plots for piece of mind. stats is for cyc=04, 11. There is a mismatch in VDATES from stats to plots b/c I have it hardcoded in the plots driver to 20230908.

STATS
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_nam_firewxnest_grid2obs_stats_00.o9651187*`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/cam/atmos.20230910/nam_firewxnest/grid2obs`

PLOTS
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_nam_firewxnest_grid2obs_plots.o96512105`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/cam/atmos.20230908`

**Comment by PerryShafran-NOAA** [comment]
Plots should be 1 day later than stats because plots is run after 00Z.  I think that's what I had but I may have been mistaken.

**Comment by ShelleyMelchior-NOAA** [comment]
I think I hard coded to an earlier date bc I ran this last night after the GMT date change. I can remove the hardcoded VDATE if you insist, but I think it is academic. The process works w/ the removal of FIXevs, which is the focus of the re-run.

File: dev/drivers/scripts/cam/plots/jevs_cam_nam_firewxnest_grid2obs_plots.sh (hunk 1 above)
Comment (PerryShafran-NOAA) [reply dev/drivers/scripts/cam/plots/jevs_cam_nam_firewxnest_grid2obs_plots.sh]: Done.

**Comment by PerryShafran-NOAA** [comment]
I think that we are good for now.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
The code review looks good. This PR is ready for merge.

**Pull request merged**